### DISCUSSION TITLE
503 Error when requesting the APIs

### DISCUSSION THREAD

**Discussion by jesteban11** [realworld-apps/realworld#1390] [DEMO / API health status] [answered]
Hi guys!
Not sure if this is the right place to check the app status. 
I was trying to get 
`curl --location --request GET 'https://api.realworld.io/api/articles/Test7-53573'` and I'm getting a 503 error
![image](https://github.com/gothinkster/realworld/assets/45263637/8186e91b-8da0-47ed-ab63-3fa4322096c7)
Is there any issue with the app?
Thanks in advance

**Comment by geromegrignon** [comment]
Hi, again Heroku messing up.

I'll investigate about alternatives but as the project is not sponsored, it might take some time to find the less costly solution for me.

**Reply by geromegrignon** [reply, marked as answer]
I triggered a manual deployment meanwhile, and it's back online. 'Weirdly' It happened just before a Heroku update on their dashboard.

**Reply by jesteban11** [reply]
I've just validated that the API is working now! Thanks!
